crates/contracts/solidity/PhonyERC20.sol

```diff
@@ -25,6 +25,8 @@ contract PhonyERC20 {
     /// addresses up to 0xffff are reserved for pre-compiles.
     address constant private IMPLEMENTATION = address(0x10000);
 
+    event Transfer(address indexed from, address indexed to, uint256 value);
```

Comment:
Slightly unrelated change - but it makes the Tenderly simulations look nicer and include UI elements for the "internal" token transfers as well.